spec/models/review_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe Review, type: :model do
+  subject do
+    bacon = Item.new(
+      name: "bacon",
+      sell_in: 4,
+      quality: 10,
+      description: "meat"
+    )
+
+    described_class.new(
+      item: bacon,
+      text: "delicious meat"
+    )
+  end
+
+  it "is valid with valid attributes" do
```

Comment:
I would put `described_class.new` into this test itself and maybe put `Item.new` into a `before_each()` block or helper function. After all, you are asserting on the described class in this test alone - the other test should be asserting on a different review with different data.

Reply:
resolved